### DISCUSSION TITLE
How to use auto generated early bound files as constants for column set?

### DISCUSSION THREAD

**Discussion by caitanya-3c** [microsoft/PowerPlatform-DataverseServiceClient#434] [Q&A] [answered]
Is it possible when using column set to use the values from an auto generated file without using nameof()? I have seen most examples use the organization service context and linq to make use of the file, but is there a way to use the strongly typed values to specify in a column set instead?

**Comment by MattB-msft** [comment]
You would use pac modelbuilder build --help to see the properties you can pass to create strongly types classes. 
One of the options creates a field list constant class.   You can store the config to do this and regenerate it at will. 

The XrmToolbox tool called "EarlyBoundGenerator" also offers a UX layer and some enhancements on pac modelbuilder if you prefer to use a UX tool.

**Reply by JimDaly** [reply, marked as answer]
I believe this is the [--emitfieldsclasses](https://learn.microsoft.com/en-us/power-platform/developer/cli/reference/modelbuilder#--emitfieldsclasses--efc) parameter.

**Reply by caitanya-3c** [reply]
Yep that's the one thank you
